Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1274,19 +1257,23 @@ def get_remote_data_command(service: client.Service, args: dict, close_incident,
             updated_notable.get("owner")) if mapper.should_map else updated_notable.get("owner")
 
     demisto.debug('notable {} data: {}'.format(notable_id, updated_notable))
-    if updated_notable.get('status') == '5' and close_incident:
-        demisto.info('Closing incident related to notable {}'.format(notable_id))
-        entries = [{
-            'Type': EntryType.NOTE,
-            'Contents': {
-                'dbotIncidentClose': True,
-                'closeReason': 'Notable event was closed on Splunk.'
-            },
-            'ContentsFormat': EntryFormat.JSON
-        }]
+    if close_incident and updated_notable.get('status_label') is not None:
+        status_label = updated_notable.get('status_label')
+        # If `status_label` in the list of status labels to close, or if 'status_end' is True,
+        # and the special label is used, close the incident on XSOAR.
+        if status_label == "Closed" or (status_label in close_extra_labels) \
+                or (close_end_statuses and updated_notable.get('status_label') in close_extra_labels):
```

Comment:
Regarding: `close_end_statuses and updated_notable.get('status_label') in close_extra_labels`
We didn't say that if `close_end_statuses = True` so we need to check that the `updated_notable.get('status_label')` is from `close_status`?

Reply:
You're right, I the condition got messed up with a commit I reverted, it should be:
```python
if status_label == "Closed" or (status_label in close_extra_labels) \
        or (close_end_statuses and argToBoolean(updated_notable.get('status_end', 'false'))):
```

I'll push a fix.